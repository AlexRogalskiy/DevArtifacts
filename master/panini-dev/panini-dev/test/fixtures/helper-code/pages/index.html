{{#code 'css'}}
.good-design {
  font-weight: 100;
  color: #000;
  box-shadow: 0 -1px 0 #fff;
}
{{/code}}
