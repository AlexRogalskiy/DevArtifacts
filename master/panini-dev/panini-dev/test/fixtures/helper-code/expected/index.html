<html>
  <body>
    <div class="code-example"><pre><code class="css"><span class="hljs-class">.good-design</span> <span class="hljs-rules">{
      <span class="hljs-rule"><span class="hljs-attribute">font-weight</span>:<span class="hljs-value"> <span class="hljs-number">100</span></span></span>;
      <span class="hljs-rule"><span class="hljs-attribute">color</span>:<span class="hljs-value"> <span class="hljs-hexcolor">#000</span></span></span>;
      <span class="hljs-rule"><span class="hljs-attribute">box-shadow</span>:<span class="hljs-value"> <span class="hljs-number">0</span> -<span class="hljs-number">1px</span> <span class="hljs-number">0</span> <span class="hljs-hexcolor">#fff</span></span></span>;
    }</span>
    </code></pre></div>
  </body>
</html>
